components/dropdown/dropdown-menu.js

```diff
@@ -12,6 +15,19 @@ import { ThemeMixin } from '../../mixins/theme/theme-mixin.js';
  */
 class DropdownMenu extends ThemeMixin(DropdownContentMixin(LitElement)) {
 
+	static get properties() {
+		return {
+			/**
+			 * Close dropdown when radio is pressed.
+			 * @type {boolean}
+			 */
+			closeRadio: {
```

Comment:
Since the styles were moved into this file then this can be treated as an internal property (e.g., [Daylight docs](https://daylight.d2l.dev/developing/best-practices/attributes-properties/#do%3A-use-a-leading-underscore-for-private-properties)). Would end up more as:
```
_closeRadio: { type: Boolean, reflect: true, attribute: '_close-radio' }
```
and you would remove the description.

Just as an FYI for future work - the comments above the property description are used to generate the docs for the Daylight site. For properties where we wouldn't want consumers to actually use them, such as this one, we would instead set the description to `@ignore`. If you do add an underscore in front of the property then you don't have to worry about that, as those properties get ignored by Daylight.